src/hooks/economics/useGetEgldPrice.ts

```diff
@@ -0,0 +1,25 @@
+import axios from 'axios';
+import useSwr from 'swr';
+
+import { EconomicsInfoType } from 'apiCalls';
+import { ECONOMICS_ENDPOINT } from 'apiCalls/endpoints';
+import { useSelector } from 'reduxStore/DappProviderContext';
+import { apiAddressSelector } from 'reduxStore/selectors';
+
+export function useGetEgldPrice() {
+  const apiAddress = useSelector(apiAddressSelector);
+
+  const fetcher = (url: string) =>
+    axios
+      .get<EconomicsInfoType>(url, {
+        baseURL: apiAddress
```

Comment:
Duplicate code. Maybe can be extracted into `getEconomics`/`fetchEconomics` function and used in both places (here and in `getEconomicsInfo`)